modules/dynamic-forms/src/components/utils.tsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { ReactElement } from "react";
+
+import { FieldMetaState, FieldProps, useField } from "react-final-form";
+import { FormHelperText, FormHelperTextProps } from "@oxygen-ui/react";
+import DynamicFormField from "./dynamic-form-field";
+
+export interface ErrorMessageProps {
+	showError: boolean;
```

Comment:
Check the indentation.

Reply:
The indentation is 1 tab space in the file as required. Seems like and issue with the way it's displayed here.